rasa/utils/tensorflow/models.py

```diff
@@ -109,12 +142,17 @@ def fit(
                         epoch_batch_size,
                         False,
                     )
-                    val_results = self._get_metric_results(prefix="val_")
+                    val_results = self._get_metric_results(
+                        epoch, self.test_summary_writer, prefix="val_"
```

Comment:
We don't need the prefix, if we are creating the a separate summary writer for validation set. I think it's definitely better to create a separate summary writer for validation set as you can overlay train and val metrics then. so lets remove the `val_` prefix.